components/Home/Hero/index.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import Hero from '.';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof Hero>;
+type Meta = MetaObj<typeof Hero>;
+
+export const Default: Story = {
+  args: {
+    title: 'Run JavaScript Everywhere.',
+    subTitle:
+      'Node.js is a free, open-sourced, cross-platform JavaScript run-time environment that lets developers write command line tools and server-side scripts outside of a browser.',
+    nodeReleaseData: [
```

Comment:
this will also use the fixture

Reply:
Is there any reason for this. This is being used only in single test unlike NodeReleaseData?

Reply:
we should create an fixture to gather all thing in one point.